src/BPOTF/OBPOTF.h

```diff
@@ -39,63 +38,145 @@ class OBPOTF
    py::array_t<uint8_t> (OBPOTF::*m_pf_decoding_func)(py::array_t<uint8_t, C_FMT> const &);
 
    /********************************************************************************************************************
     * PUBLIC MEMBER VARIABLE DECLARATION 
     *******************************************************************************************************************/
    public:
+
+   /********************************************************************************************************************
+    * @typedef ECodeType_t
+    * @brief   This typedef is an enumeration type that indicates the different codes that are supported for 
+    *          decodification.
+    *******************************************************************************************************************/
    typedef enum 
    {
-      E_GENERIC   = 0,  // Default mode
-      E_CLN       = 1   // Circuit-level noise codes
+      E_GENERIC   = 0,  //!< Default mode
+      E_CLN       = 1   //!< Circuit-level noise codes
    } ECodeType_t;
 
    /********************************************************************************************************************
     * PRIVATE CLASS METHOD DECLARATION
     *******************************************************************************************************************/
    private:
+
+   /********************************************************************************************************************
+    * @brief This routine performs the OTF algorithm using the clasical Unified-Find method.
+    * 
+    * @param llrs[in]   The llrs is a vector containing the probabilities of the PCM columns.
+    * @return std::vector<uint64_t> The return value is a vector containing the recovered error.
+    *******************************************************************************************************************/
    std::vector<uint64_t> otf_classical_uf(std::vector<double> const & llrs);
 
+   /********************************************************************************************************************
+    * @brief This routine performs the OTF algorithm.
+    * 
+    * @param llrs[in]   The llrs is a vector containing the probabilities of the PCM columns.
+    * @return std::vector<uint64_t> The return value is a vector containing the recovered error.
+    *******************************************************************************************************************/
    std::vector<uint64_t> otf_uf(std::vector<double> const & llrs);
 
+   /********************************************************************************************************************
+    * @brief This routine returns a vector of the sorted indexes based on the probabilities of the llrs. It copies the 
+    *        initial vector with the unsorted indexes from the member variable m_au64_index_array.
+    * 
+    * @param llrs[in]   The llrs is a vector containing the probabilities of the PCM columns.
```

Comment:
- "based on the probabilities of the llrs" --> "based on the magnitude of the llrs"
- the llrs definition same as in previous two comments